tools/spelling_dictionary.txt

```diff
@@ -549,6 +550,7 @@ filename
 filenames
 fileno
 filesystem
+filterchain
```

Comment:
nit: I would remove this and just use "filter chain" in whatever comment this is failing on.

Reply:
done